### PR TITLE
DWR-845, initial changes for district/school groups

### PR DESCRIPTION
**Pull request by malaffoon**
Like the admin review, this is mostly changes moving shared code into RDW_Common. I'll try to call out the significant changes. Also, this isn't all the changes for district/school group -- the actual filtering still needs to be done.

### CODE REVIEW

File: common/src/test/java/org/opentestsystem/rdw/reporting/common/test/support/PermissionScopes.java [hunk 1]
```diff
@@ -41,16 +41,11 @@ public static PermissionScope statewide() {
         return PermissionScope.STATEWIDE;
     }
 
-    public static PermissionScope empty() {
-        return PermissionScope.EMPTY;
-    }
-
     public static PermissionScope districts(final Long... ids) {
-        return new PermissionScope(Arrays.asList(ids), null);
+        return new PermissionScope(null, Arrays.asList(ids), null, null);
```
Comment (malaffoon) [inline common/src/test/java/org/opentestsystem/rdw/reporting/common/test/support/PermissionScopes.java]: @esotrope i'll address the too-many-ctor-args issue in a subsequent PR.

File: webapp/src/main/java/org/opentestsystem/rdw/reporting/Application.java [hunk 2]
```diff
@@ -28,7 +29,8 @@
         YamlPropertiesConfigurator.class,
         DataSourceConfiguration.class,
         TranslationConfiguration.class,
-        ReportConfiguration.class
+        ReportConfiguration.class,
+        JdbcOrganizationRepository.class
```
Comment (malaffoon) [inline webapp/src/main/java/org/opentestsystem/rdw/reporting/Application.java]: Force repository component to be in the application.

File: webapp/src/main/java/org/opentestsystem/rdw/reporting/security/auth/DefaultAuthorizationService.java [hunk 3]
```diff
@@ -3,14 +3,14 @@
 import com.google.common.collect.ImmutableSet;
```
Comment (malaffoon) [inline webapp/src/main/java/org/opentestsystem/rdw/reporting/security/auth/DefaultAuthorizationService.java]: Most of the real changes are in here.

File: common/src/test/java/org/opentestsystem/rdw/reporting/common/test/support/PermissionScopes.java (hunk 1 above)
Comment (esotrope) [reply common/src/test/java/org/opentestsystem/rdw/reporting/common/test/support/PermissionScopes.java]: No biggy. Definitely bigger priorities.

**Pull request merged**